_schemas/1.0/tests/response/invalid/errors/invalid_error_objects.json

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
Would be great if you could add some information (e.g. as `detail` property) _why_ the error objects are invalid.

Reply:
I have added informations about the errors present in this document, but also for all the other test files. Here is an exemple for an invalid response : 
```
{
  "data": "not valid",
  "meta": {
    "errors-present-in-document": [
      {
        "detail": "Primary data **MUST** be either: 1) a single resource object, a single resource identifier object, or null, for requests that target single resource 2) an array of resource objects, an array of resource identifier objects, or an empty array ([]), for requests that target resource collections",
        "source": {
          "pointer": "/data"
        }
      }
    ]
  }
}

```